### DISCUSSION TITLE
MyTourbook show no cadence values

### DISCUSSION THREAD

**Discussion by ovahead** [mytourbook/mytourbook#933] [Q&A]
I'm not really in cadence. But my second sensor is a 2 in 1 sensor, for cadence and speed (presumably as speed sensor for my second bike). I tested it today as cadence sensor. My device showed me the cadence values but there not shown in Mytourbook. Not in Tour Info or even in Tour Analysis. Is it probably a (known) bug?

BTW
The sensor is normal listed in the Tour Info as Magene - 112 with battery voltage.

**Comment by ovahead** [comment]
Okay, I found out that this first must be activated in the Tour Info. Very tricky! ;)

**Reply by FJBDev** [reply]
If you are new to MyTourbook, I highly recommend to read the [documentation](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation)

**Reply by ovahead** [reply]
Thanks! I know the documentation already. The problem is there isn't a search function as far as I know.

**Reply by FJBDev** [reply]
> The problem is there isn't a search function as far as I know.

What you can do is this type of search in google, it should help narrowing down what you are looking for

`tour chart cadence site:mytourbook.sourceforge.io/mytourbook/`

**Reply by ovahead** [reply]
Thanks, Frederic!
I will try this in the future.
